### ISSUE TITLE
Memory leak when creating thumnails of webp pages

### ISSUE THREAD

**Issue by sebdelsol** [beville/ComicStreamer#1]
[tested on windows 7]
Memory usage increases for each thumbnail created in  ImageAPIHandler.resizeImage() when the cover is a webp image.
Guess it's caused by [this Pillow bug](https://github.com/python-pillow/Pillow/issues/586).
here is a [workaround](http://stackoverflow.com/questions/21590496/using-webp-images-with-pillow-2-2-2-or-2-3-0).

**Comment by beville** [comment]
Good catch!  And quite a bummer.  I'm going to have to look into this more, to understand which layer the leak is happening at.  This could be a big pain in the tuchus to manage, depending.

**Comment by sebdelsol** [comment]
Pain in the tuchus is quite the euphemism.
This workaround needs you to compile and distribute the libwebp.dll and link with the standard libc or you'll have to distribute the visual studio runtime dll too, so you won't end up with an unstable malloc/free pair.

tl;dr: I'm not sure it'll be worth the effort. You'd rather wait for Pillow to fix their bug, and skip wepb cover's thumnails at the moment.

**Comment by beville** [comment]
Per your suggestion, thumbnails of WebP covers are skipped in the lastest version. Let's leave this open until Pillow addresses this.

**Comment by wiredfool** [comment]
There's a potential fix for this in Pillow master now. I'd appreciate reports of success/failure.
